client/src/lib/components/DatasetForm/DatasetForm.svelte

```diff
@@ -15,7 +17,10 @@
     description: "",
     formats: [],
     entrypointEmail: "",
-    contactEmails: [],
+    contactEmails: [$user?.email] || [""],
```

Comment:
Je n'avais pas vu ça : attention, si `$user` est `null` alors on obtient `[null] || []` ce qui est évalué à `[null]` (en JS un array vide est truthy)

Sachant ça, je crois qu'on peut faire ça :

```suggestion
    contactEmails: [$user?.email || ""],
```